sbt_common/Makefile

```diff
@@ -34,6 +34,13 @@ sbt-messaging-publish:
 	echo "Nothing to do!"
 
 
+sbt-monitoring-test:
+	$(call sbt_test,common_monitoring)
```

Comment:
I’m moderately surprised this works – why doesn’t it whinge about the lack of a docker-compose file?